command/pr_checkout.go

```diff
@@ -38,8 +38,7 @@ func prCheckout(cmd *cobra.Command, args []string) error {
 		headRemote, _ = remotes.FindByRepo(pr.HeadRepositoryOwner.Login, pr.HeadRepository.Name)
 	}
 
-	cmdQueue := [][]string{}
-
+	cmdQueue := make([][]string, 0, 4)
```

Comment:
Did you choose 4 arbitrarily?

Reply:
Thank you for explaining. Since the connection between "4" and the number of `append`s in the following code might not be apparent a future reader of this code, and since we don't really depend on doing a runtime/memory optimization here, I would say that this optimization is premature and that it might be clearer to take this out.